css/media.css

```diff
@@ -0,0 +1,1039 @@
+
+  .copyright {
+    margin: 0;
+  }
+}
+
+@media screen and (max-width: 740px) {
+  .products__product-card:nth-child(2) {
+    display: none;
+  }
+
+  .video-block {
+    align-items: center;
+  }
+
+  .video-block__text {
+    max-width: 89%;
+    margin: 14px 0 27px 5px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .stories {
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+  }
+
+  .stories-card {
+    max-width: 90%;
+    margin: 0;
+  }
+
+  .stories-card:nth-child(2) {
+    display: none;
+  }
+
+  .stories-card__pic {
+    margin: 0 auto;
+    max-width: 100%;
+    /* min-height: 100%; */
+    display: block;
+  }
+
+  .stories-card__title {
+    font-size: 20px;
+    margin: 25px 0 7px 1px;
+  }
+
+  .stories-card__descr {
+    margin: 0 0 22px 1px;
+    font-size: 14px;
+    line-height: 18px;
+  }
+
+  .stories-card__link-more {
+    font-size: 12px;
+  }
+
+  .stories-card__link-more::after {
+    content: url(../img/Arrow-wh-375.png);
+  }
+
+  .gallery {
+    max-width: 85%;
+  }
+}
+
+@media screen and (max-width: 570px) {
+  .container {
+    max-width: 100%;
+  }
+  
+  .promo-section::after {
+    height: 318px;
+  }
+  
+  /* Footer */
+  .footer {
+    padding: 0;
+    padding-bottom: 34px;
+    height: auto;
+  }
+
+  .footer-navbar {
+    display: flex;
+    padding: 0;
+    flex-direction: column;
+    align-content: center;
+    align-items: center;
+  }
+
+  .footer-navbar__title {
+    font-size: 15px;
+    margin-bottom: 2px;
+  }
+  
+  .footer-navbar__link {
+    font-size: 15px;
+    margin-bottom: 7.5px;
+  }
+  
+  .footer-navbar__column {
+    flex-basis: 100%;
+    min-width: 218px;
+    max-width: 100%;
+    min-height: 151px;
+    margin: 0;
+    text-align: center;
+  }
+  
+  .footer-navbar__column:first-child {
+    margin-top: 36px;
+    margin-bottom: 28px;
+  }
+  
+  .footer-navbar__sale-offer {
+    width: 167px;
+    margin-top: 7px;
+    font-size: 15px;
+    line-height: 15.5px;
+  }
+
+  .footer-navbar__sign-up {
+    margin: 21px auto;
+    padding-top: 10px;
+    width: 151px;
+    min-height: 38px;
+    font-size: 12px;
+  }
+  
+  .social {
+    width: 168px;
+    align-items: center;
+    margin: 0 auto;
+    margin-top: 104px;
+  }
+  
+  .copyright {
+    margin-top: 26px;
+    font-size: 13px;
+  }
+}
+
+@media screen and (max-width: 375px) {
+  .container {
+    max-width: 375px;
+  }
+  
+  .header {
+    min-height: 53px;
+    padding: 0 28px 0 30px;
+  }
+  
+  .header__logo-mobile {
+    margin-top: 20px;
+  }
+  
+  .header__logo-mobile img {
+    width: 104px;
+  }
+  
+  .burger {
+    width: 41px;
+    height: 5px;
+    margin-top: 9px;
+  }
+  
+  .burger::after {
+    margin-top: 8px;
+    width: 41px;
+    height: 6px;
+  }
+
+  /* market-section */
+
+  .market-section {
+    padding-top: 18px;
+    padding-bottom: 100px;
+  }
+  
+  .banner {
+    padding: 50px 0 0 48px;
+    min-height: 192px;
+    background-position: 20% ;
+  }  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)